string namespace
string type
geometry_msgs/PoseStamped pose_D_D
geometry_msgs/PoseStamped pose_R_D
float64 alpha
---
string namespace
string type
geometry_msgs/PoseStamped pose_R_D
geometry_msgs/TransformStamped transform_stamped

